src/main/resources/swagger/api-docs.yaml

```diff
@@ -4115,10 +4115,30 @@ definitions:
       ok:
         type: boolean
         description: whether any system(s) need attention
-
+        default: false
       systems:
         type: object
         description: Map[String, SubsystemStatus]
+        default: {
```

Comment:
@aednichols - Thank you! My thoughts are that we should try to stick to yaml for the yaml file. It's cleaner, more concise, and consistent with the rest of the file. Previously, we've had even more json in the file and have been weeding that out over time.